fixup.sh

```diff
@@ -0,0 +1,13 @@
+cat >dist/cjs/package.json <<!EOF
```

Comment:
Could we use `echo` instead?

Reply:
Done, script is now easier to read and the output is the same